code/game/objects/structures/lavaland/necropolis_tendril.dm

```diff
@@ -36,13 +36,9 @@ GLOBAL_LIST_INIT(tendrils, list())
 	gps = new /obj/item/gps/internal(src)
 	GLOB.tendrils += src
 
-/obj/structure/spawner/lavaland/deconstruct(disassembled)
+/obj/structure/spawner/lavaland/Destroy()
```

Comment:
What if we need to destroy it with no chasm and loot?
I think here need make Destroy(force)
and if we have "force" dont spawn loot and chasm.
Or leave it as before, you make explosion dont hard qdel it.